templates/inspec/examples/google_pubsub_subscription/google_pubsub_subscriptions.erb

```diff
@@ -1,8 +1,7 @@
 <% gcp_project_id = "#{external_attribute('gcp_project_id', doc_generation)}" -%>
 <% subscription = grab_attributes['subscription'] -%>
 describe google_pubsub_subscriptions(project: <%= gcp_project_id -%>) do
-  it { should exist }
```

Comment:
Would plural resources ever have a 'should exist' field?

Reply:
No, this was a mistake. It doesn't make sense to ask if a plural resource exists, because it is a collection of individual resources. Testing on `count` makes more sense